services/proxy/main.go

```diff
@@ -23,7 +25,12 @@ type Configuration struct {
 	MaxRecipients int `yaml:"max_recipients"`
 	// YMAddr is the address of the YM entrypoint
 	YMAddr string `yaml:"yulmails_address"`
-	// TODO: add TLS configuration
+	// Enable tls : true/false
```

Comment:
following Go standard for an exported variable: comments must match the exact variable name or function name
```suggestion
	// EnableTLS will activate or not TLS
```